lib/llm-events/aws-bedrock/chat-completion-summary.js

```diff
@@ -36,7 +36,7 @@ class LlmChatCompletionSummary extends LlmEvent {
     const cmd = this.bedrockCommand
     this[cfr] = this.bedrockResponse.finishReason
     this[rt] = cmd.temperature
-    this[nm] = 1 + this.bedrockResponse.completions.length
+    this[nm] = (this.bedrockCommand.prompt?.length ?? 0) + this.bedrockResponse.completions.length
```

Comment:
based on the `bedrockCommand` file this is always an array. so if the length is 0 it'll be 0, do we need this defaulting to 0 when prompt.length is falsey?